docs/src/pages/demos/lists/CheckboxList.js

```diff
@@ -44,7 +44,13 @@ class CheckboxList extends React.Component {
       <div className={classes.root}>
         <List>
           {[0, 1, 2, 3].map(value => (
-            <ListItem dense button key={value} onClick={event => this.handleToggle(event, value)}>
+            <ListItem
+              key={value}
+              dense
+              button
+              onClick={this.handleToggle(value)}
```

Comment:
shouldn't that be wrapped in an anonymous function?

Reply:
no, it shouldn't, handleToggle now returns a function

Reply:
@cherniavskii It's a micro-optimization. I think that calling a function is faster than creating one.

Reply:
@oliviertassinari probably you are right